linetools/analysis/absline.py

```diff
@@ -531,3 +567,47 @@ def Wr_from_N_transition(N, transition, llist='ISM'):
     # return
     return Wr_from_N(N, wrest, fosc)
 
+
+def N_from_Wr_transition(Wr, transition, llist='ISM'):
```

Comment:
I would rather not overload this the input statement as it goes somewhat against my preferred direction when coding. Instead, I have added the corresponding method to the AbsLine class, so you can use it directly from there. I hope you agree with this decision.